packages/core/tests/integration/components/navi-visualization-config/line-chart-test.ts

```diff
@@ -85,8 +83,8 @@ module('Integration | Component | visualization config/line chart', function(hoo
       .dom('.line-chart-config .line-chart-config__series-config')
       .isVisible('The series config component is rendered when `stacked` option is on');
 
-    await click('.line-chart-config__stacked-opt');
-    assert.dom('.line-chart-config__stacked-opt').isNotChecked('Stacked it not true');
+    await click('.line-chart-config__stacked-opt-select');
+    assert.dom('.line-chart-config__series-config').isVisible('Stacked it not true');
```

Comment:
Why did this change? Also can we update the assert message

Reply:
would having it be stacked-opt-toggle make more sense than select? think the class name just changed slightly. As far as the test goes, the animation wasnt triggering in time for the assert to see that the toggle is actually checked. I tried to add in an await animationsSettled, but the import file was unfound